packages/core/src/linker/compiler.ts

```diff
@@ -67,7 +67,7 @@ export class Compiler {
   }
 
   /**
-   * Same as {@link #compileModuleSync} but also creates ComponentFactories for all components.
+   * Same as {@link Compiler#compileModuleSync} but also creates ComponentFactories for all components.
```

Comment:
Actually we'll need to go one step further: 
```suggestion
   * Same as {@link Compiler#compileModuleSync compileModuleSync} but also creates ComponentFactories for all components.
```

Reply:
Fixed it.
Didn't even think of that, is it to display the link name correctly?
Thank you!